tensorflow/lite/micro/examples/memory_footprint/create_adder_model.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Creates a simple tflite model that adds two input tensor of size 1."""
+
+from absl import app
+import tensorflow as tf
+
+
+def main(_):
+  input_shape = (1, 1, 1)
+  x1 = tf.keras.layers.Input(input_shape)
+  x2 = tf.keras.layers.Input(input_shape)
+
+  # equivalent to `added = tf.keras.layers.add([x1, x2])`
+  added = tf.keras.layers.Add()([x1, x2])
+  model = tf.keras.models.Model(inputs=[x1, x2], outputs=added)
+  model.save("simple_add_model")
+
+  converter = tf.lite.TFLiteConverter.from_saved_model("simple_add_model")
```

Comment:
do we need to go to a saved_model? can we directly go from keras to tflite, similar to https://github.com/tensorflow/tflite-micro/blob/ca14acc51a92912da5b18da3f97391d773842925/tensorflow/lite/micro/testing/generate_test_models.py#L64-L65

Reply:
Done. Thanks for pointing this to me!